src/PumpStream.php

```diff
@@ -18,8 +18,8 @@
  */
 class PumpStream implements StreamInterface
 {
-    /** @var callable */
-    private $source;
+    /** @var callable|null */
+    private $source = null;
```

Comment:
Note that I changed this.. can be null upon detach..

Reply:
👍

Reply:
Resolving